dipu/torch_dipu/csrc_dipu/runtime/distributed/distributedUtil.h

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) 2024, DeepLink.
+#pragma once
+
+#include "csrc_dipu/runtime/core/DIPUGuard.h"
+
+#include "DICLUtils.hpp"
+
+namespace dipu {
+
+class distributedUtil {
```

Comment:
```suggestion
class DistributedUtil {
```

仅仅为了 ascend allreuduce 搞出这个抽象其实还是挺奇怪的，鉴于应用范围还很窄，或者加点注释之类的，或者考虑一下名字问题。